crates/bevy_ecs/src/storage/mod.rs

```diff
@@ -1,4 +1,24 @@
 //! Storage layouts for ECS data.
+//!
+//! This module contains the low level backing data stores for ECS. These all offer minimal and often
+//! unsafe APIs, exposed primarily for debugging and monitoring purposes.
+//!
+//! # Fetching Storages
+//! Each of the primary backing data stores can be fetched via [`Storages`], which can be fetched from a
```

Comment:
What is a “primary backing datastore”? Is there a secondary backing data store?

Reply:
That is a good point, there are no secondary data stores. Should I just remove that word?

Reply:
Yeah, I think that makes sense. Just do something like... 

```suggestion
//! Each of the below data stores can be fetched via [`Storages`], which can be fetched from a
```